lib/chef/knife/knife.rb

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright:: Copyright (c) 2014 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'chef-analytics/identity'
+require 'chef/config'
+
+module KnifeAnalytics
```

Comment:
Could we make this a module that we include rather than inherit from this?  One advantage of inheriting from Chef::Knife directly is that it helps keep `knife rehash` working correctly.